app/models/user.rb

```diff
@@ -1,10 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
Kindly make sure that the user is also able to confirm his account. For this kindly add the `:confirmable` attribute to the devise attribute list so that application asks for confirmation of the email.